pkg/ruler/compat.go

```diff
@@ -330,7 +330,7 @@ func DefaultTenantManagerFactory(
 			ForGracePeriod:             cfg.ForGracePeriod,
 			ResendDelay:                cfg.ResendDelay,
 			AlwaysRestoreAlertState:    true,
-			DefaultEvaluationDelay: func() time.Duration {
+			DefaultRuleQueryOffset: func() time.Duration {
 				// Delay the evaluation of all rules by a set interval to give a buffer
 				// to metric that haven't been forwarded to Mimir yet.
 				return overrides.EvaluationDelay(userID)
```

Comment:
nit [non-blocking]: At some point, we probably want to create a new limit so that it matches the description a bit more accurately. Happy to create an issue for it if you think it's worth it.

Reply:
I agree we should but looks very low priority. I'm fine creating an issue for now.